README.md

```diff
@@ -12,7 +12,13 @@
 
 # Kinvey CLI
 
-Kinvey CLI is a utility for deploying and managing FlexServices running on the Kinvey FlexService Runtime.
+Kinvey CLI is a utility for managing various aspects of your Kinvey account from the command line. Its features include:
+
+* Deploying and managing FlexServices running on the Kinvey FlexService Runtime
+* Viewing organizations and setting a default organization
```

Comment:
Setting default/active organization is in the scope of the CLI only. Seeing it mentioned here makes me think that it's a change that happens in the backend. Not sure if it is just me...

Reply:
You are right. Organization selection only makes sense under Managing Flex so I'm deleting this bullet.